examples/qtensor/qtensor.morpho

```diff
@@ -62,7 +67,7 @@ var bulk = AreaIntegral(landau, q_tensor)
 // Similarly, the anchoring free energy is now the line-integral of the anchoring functional with the field input being q_tensor
 var anchor = LineIntegral(anchoring, q_tensor)
 
-// The elastic energy in just the grad-squared of the q-tensor
+// The elastic energy in the grad-squared of the q-tensor
```

Comment:
// The elastic energy ~~in~~ __is__ the grad-squared of the q-tensor

Reply:
Dang :/
Fixed it now.